xdsl/dialects/csl.py

```diff
@@ -376,6 +376,14 @@ class ImportModuleConstOp(IRDLOperation):
 
     result = result_def(ImportedModuleType)
 
+    @staticmethod
+    def get(name: str, *params: SSAValue | Operation) -> ImportModuleConstOp:
+        return ImportModuleConstOp(
+            operands=[[]] if len(params) == 0 else params,
+            result_types=[ImportedModuleType()],
+            properties={"module": StringAttr(name)},
+        )
```

Comment:
I would prefer to see this as a custom `__init__` instead of a `.get`. We are trying to move away from these.